Automattic/studio
# Remove arrow from installer background

Related to https://github.com/Automattic/dotcom-forge/issues/7004

## Proposed Changes

- Removes the arrow from the left side of the installer background.

| Before | After |
| ------- | ------- |
| <img width="822" alt="Screenshot 2024-05-10 at 08 55 27" src="https://github.com/Automattic/studio/assets/417538/f83e6f69-c1fc-4a44-8f64-74691b9d72cb"> | <img width="822" alt="Screenshot 2024-05-10 at 08 54 10" src="https://github.com/Automattic/studio/assets/417538/d23030c4-6e6f-4f62-932f-dfe5c32b5621"> |

## Testing Instructions



- Run `rpm run make`
- Open `out/make/Studio-*.dmg`
- Note that the arrow icon is no longer displayed

## Pre-merge Checklist
